Assets/MRTK/Examples/Demos/README.md

```diff
@@ -49,3 +49,8 @@ Several mechanisms to orientate and place object in a scene relative to each oth
 Several default line drawing options for use in your project. Also utilized by the MRTK itself for controller pointer and teleportation use.
 
 > Keep checking back often as more UX controls are added.
+
+## Projection Options
+
+* Projection Options Demo
+Demonstrates how to alter the HoloLens2 field of view to trade view size for detail.  This may be helpful when small details need to be visualized on HoloLens2.  There is also the ability to change the Render Viewport Scale which may increase FPS by having the application render fewer pixels.
```

Comment:
Similar comment to the one above, would it be worth linking to the docs on reading mode:

https://docs.microsoft.com/en-us/hololens/hololens2-display

(I think that we'd probably need up update that doc to have a link-able part, but even without that it might be mentioning the doc)